pkg/mongodb/MongodbMessage.php

```diff
@@ -65,6 +65,22 @@ public function __construct(string $body = '', array $properties = [], array $he
         $this->redelivered = false;
     }
 
+    public static function fromArrayDbResult(array $arrayResult): self
```

Comment:
move the method to the context, make it public with `@internal` docblock, look at AMQP cotext

Reply:
```suggestion
    public static function fromArrayDbResult(array $arrayResult): self
```

Reply:
ok, done!